lib/docassemble/takedown.js

```diff
@@ -18,6 +18,7 @@ const takedown = async () => {
     log.info({ type: `takedown`, pre: `Success! Test takedown has finished successfully.` });
   } catch (error) {
     log.error({type: `takedown`, pre: `could not delete project`, data: JSON.stringify(error, null, 4)})
+    // we don't re-throw the exception, so make sure to exit with 1 to let Github Actions know the script failed
```

Comment:
```suggestion
    // we don't re-throw the exception to avoid an "unhandled promise rejection", so make sure to exit with 1 to let Github Actions know the script failed
```

Reply:
I'm trying out putting the note about the error right where the weird looking code is. You can take it or leave it.

Reply:
Thanks, but I think I'll pass: there's more clarification at the[level of the try catch](https://github.com/SuffolkLITLab/ALKiln/pull/632/files#diff-a424932832259b654fe8ce9283c6c61a8c7cc72e07daccff8c0488974557af15R13) just a few lines higher. The try-catch is the specific thing that's stopping the unhandled promise rejection, so I'd prefer to keep that comment closer to it.